docs/how-to-use-runhouse.rst

```diff
@@ -12,8 +12,8 @@ Before reviewing this detailed guide, we recommend you start with the `Quick Sta
 * Install Runhouse with ``pip install runhouse``
 
 * Optionally install with a specific cloud like ``pip install "runhouse[aws]"`` or with
-  `SkyPilot <https://skypilot.readthedocs.io/en/latest/docs/index.html>`_ for elastic compute
-  ``pip install "runhouse[sky]"``
+  `SkyPilot <https://skypilot.readthedocs.io/en/latest/docs/index.html>`_ for elastic compute with
```

Comment:
after some docs refactoring, we now have
* more comprehensive install + compute setup page
* how to use runhouse -> development guide (directing to installation/quick start for prereqs)

I think we should update this in the installation page. and in the quick-start-local page (standalone, not in the table of contents but is linked out from the installation page) we should update `pip install runhouse` to `pip install runhouse[server]`